io/src/test/scala/sbt/io/CopyDirectorySpec.scala

```diff
@@ -11,11 +11,10 @@
 package sbt.io
 
 import java.nio.file._
-import org.scalatest.diagrams.Diagrams
 import org.scalatest.flatspec.AnyFlatSpec
 import sbt.io.syntax._
 
-class CopyDirectorySpec extends AnyFlatSpec with Diagrams {
+class CopyDirectorySpec extends AnyFlatSpec {
```

Comment:
diagram assert does not work with Scala 3 (compile error) 😢

Reply:
Is it a known bug in ScalaTest?